api/cluster.proto

```diff
@@ -20,17 +20,19 @@ message DescribeSubnetsRequest {
 	uint32 offset = 3;
 	google.protobuf.UInt32Value subnet_type = 4;
 	repeated string subnet_id = 5;
-	repeated string advanced_param = 6;
+	repeated string zone = 6;
```

Comment:
Is there zone in runtime？

Reply:
We need a field to determine whitch availabilityZone to deploy to. It just like we have pek3 region, then I must choice a availabilityZone to deploy my cluster.